route/src/route.cpp

```diff
@@ -53,6 +56,7 @@ namespace route {
         pnh_->getParam("route_file_path", route_file_location);
         rg_worker_.set_route_file_path(route_file_location);
         rg_worker_.set_publishers(route_event_pub_, route_state_pub_, route_pub_,route_marker_pub_);
+        rg_worker_.set_publishers(wml_);
```

Comment:
I highly doubt this line works the way you think it does. The wml_ object is not a ros::publisher. I'm pretty amazed this even compiles.

Reply:
It shouldn't be there. Came from somewhere.